### ISSUE TITLE
An accessibility issue using mini-pager option with views in UI-Kit Starter theme

### ISSUE THREAD

**Issue by AtamDhillon** [govCMS/govcms8_uikit_starter#130]
Hi team,
We recently had an accessibility audit that picked up an issue with Pagination links block using views through UI-Kit Starter theme.

Issue - the aria-labelledby attribute for <nav> does not have corresponding "id" assigned to the relevant heading element.

`<nav class=""pager"" aria-labelledby=""pagination-heading"">`
`<h4 class=""pager__heading visually-hidden"">Pagination</h4>"`

However it seems the issue is only if we use the option as "Paged output, mini pager" to display pagination links. If we switch to "Paged output, full pager", the problem goes away. That is how we have rectified it currently without tempering the templates hoping it will be fixed at the theme level.

We wanted to bring this to your attention and possibly a fix can be implemented. Please let us know if you need more information.
